pom.xml

```diff
@@ -19,6 +19,11 @@
             <comments>All source code is under the MIT license.</comments>
         </license>
     </licenses>
+    
+    <properties>
+      <jenkins.version>1.625</jenkins.version>
+      <java.level>6</java.level>
+    </properties>
```

Comment:
Jenkins 1.625 and JDK 6? Weird. https://jenkins-ci.org/blog/2015/04/06/good-bye-java6/